docs/ldml/tr35-messageFormat.md

```diff
@@ -2,7 +2,7 @@
 
 # Unicode Locale Data Markup Language (LDML)<br/>Part 9: Message Format
 
-|Version|47 (draft)              |
+|Version|46.1 (draft)            |
```

Comment:
This was unnecessary and actually counterproductive. The version on the maint/maint-46 branch, the one targeted for 46.1, already says "46.1 (draft)", so this change may cause a conflict there when cherry-picked.  This version on main is for 47, so we will have to revert this change when we update the spec on main again.